app/models/manageiq/providers/oracle_cloud/inventory.rb

```diff
@@ -2,4 +2,17 @@ class ManageIQ::Providers::OracleCloud::Inventory < ManageIQ::Providers::Invento
   require_nested :Collector
   require_nested :Parser
   require_nested :Persister
+
+  def self.default_manager_name
+    "CloudManager"
+  end
+
+  def self.parser_classes_for(ems, target)
+    case target
+    when InventoryRefresh::TargetCollection
+      [ManageIQ::Providers::OracleCloud::Inventory::Parser]
```

Comment:
Usually this is a list of all the managers' parsers but since we're using combined manager refresh here that isn't needed, follow-up to see if we can make this simpler in core without a child provider override